### DISCUSSION TITLE
This beautiful project is active?

### DISCUSSION THREAD

**Discussion by ghost** [RomanKubiak/ctrlr#590] [Q&A] [answered]
Linux Mint 21.2_x64 Cinnamom - Ctrlr-x86_64-6.0.35.sh

Hello, I am a new user of Ctrlr and this "forum". Is this project still active? 
I have some synthesizers (hardware) (The Yamaha Tx81z is the last) that I would like to check out via Ctrlr on linux and it would be amazing if you could continue. I installed the Ctrlr-x86_64-6.0.35.sh version but when I start the program, nothing opens. Is there a log file with which I can give more information? Thanks for now...

**Comment by dnaldoog** [comment, marked as answer]
Hi @CodesoundR 

You need version [5.3.201.sh](https://sourceforge.net/projects/ctrlr-utilities/files/Ctrlr/Ctrlr-x86_64-5.3.201.sh/download) for Linux!

You can download a panel for the [TX81Z](https://sourceforge.net/projects/ctx81z/files/src/) - download the .bpanelz file for Linux.

**Comment by ghost** [comment]
Hi, thanks for your help!
Now Ctrlr works into my system. I have problems with the vst but I open another thread..... Thanks a lot again!

**Comment by damiensellier** [comment]
Hi, this project is alive and kicking, thanks to all the member of the community on this Ctrlr Github discussion board.
Good luck with your projects!
